android/src/main/java/com/android/simplSdk/SimplSdkModule.java

```diff
@@ -26,61 +26,98 @@ public SimplSdkModule(ReactApplicationContext reactContext) {
         super(reactContext);
     }
 
-    @Override
-    public String getName() {
-        return "SimplSdk";
+    @ReactMethod
+    public void isUserApproved(final ReadableMap map, final Callback successCallback, final Callback errorCallback){
+        final HashMap<String, String> approvalMap = MapUtils.toHashMap(map);
+        final HashMap<String, String> paramsMap = MapUtils.stringToHashMap(String stringMap);
+
+        try {
+            Simpl.init(getReactApplicationContext(), approvalMap.get("merchantId"));
+            if (approvalMap.get("isSandbox") == "true")
+                Simpl.getInstance().runInSandboxMode();
+
+            SimplUser user = new SimplUser(approvalMap.get("email"), approvalMap.get("phone_number"));
+            Simpl.getInstance().isUserApproved(user)
+                .addParam("transaction_amount_in_paisa", paramsMap.get("amount_in_paisa"))
```

Comment:
`transaction_amount_in_paise` and not `transaction_amount_in_paisa`

Reply:
this was a typo in previous api.